packages/react-ui-core/src/Gmap/withGoogleScript.js

```diff
@@ -0,0 +1,92 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import autobind from 'autobind-decorator'
+import scriptjs from 'scriptjs'
+import { getDisplayName } from 'recompose'
```

Comment:
how much does `recompose`+dependencies add to the bundle? If we're just using `getDisplayName`, you might just want to copy that function over and use it, since it's [fairly small](https://github.com/acdlite/recompose/blob/master/src/packages/recompose/getDisplayName.js).

You might also want to consider doing:
`import getDisplayName from 'recompose/getDisplayName`
and compare bundle sizes, see if that cuts it down.